collectors/pod.go

```diff
@@ -73,6 +73,12 @@ var (
 		[]string{"namespace", "pod"}, nil,
 	)
 
+	descPodScheduled = prometheus.NewDesc(
+		"kube_pod_scheduled",
```

Comment:
As this comes from information under the status subresource convention is to name this `kube_pod_status_...`, I'd prefer `kube_pod_status_scheduled_time`.